### PR TITLE
Update to capacitor 6

### PR DESCRIPTION
**Pull request by manusreload**
Hello,

Following https://capacitorjs.com/docs/updating/plugins/6-0, I update plugin to Capacitor 6.

Cheers!

### CODE REVIEW

File: android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
   <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-      package="com.timozacherl.plugins.CapSiriShortcuts">
```
Comment (lovetodream) [inline android/src/main/AndroidManifest.xml:3]: Has this been removed by the automatic migration?

**Review by lovetodream — COMMENTED** [review]
Looks good, just one small question inline. Sorry for keeping you waiting and thanks for the effort :)

**Comment by lovetodream** [comment]
One additional note, can you please add a disclaimer for v6 to the breaking changes section in the readme as done with all the previous versions?

File: android/src/main/AndroidManifest.xml (hunk 1 above)
Comment (manusreload) [reply android/src/main/AndroidManifest.xml:3]: Yes, package name has been moved to the build.gradle: https://github.com/lovetodream/capacitor-plugin-siri-shortcuts/pull/58/files/9e2d636f2e98dc2e01ae2aec0ce24ad58ce01bb8#diff-197b190e4a3512994d2cebed8aff5479ff88e136b8cc7a4b148ec9c3945bd65aR21

**Comment by manusreload** [comment]
Hello, I glad inform to you that the README.md was updated following the same structure that was used in previous versions of the software. Cheers!

**Review by lovetodream — APPROVED** [review]

**Comment by lovetodream** [comment]
Looks good, thank you for doing this!

**Pull request merged**